harness/helpers.ts

```diff
@@ -442,6 +442,15 @@ export class CloudTestClient extends BaseTestClient {
     }
 }
 
+export const expectErrorToContain = (message: string, expectedStrings: string[]) => {
+    expectedStrings.forEach((expected) => {
+        if (!message.includes(expected)) {
+            console.warn(`Expected error message to contain: \n ${expected}, but got: \n ${message}`)
+        }
+        expect(message).toContain(expected)
```

Comment:
there isnt much point in doing a warning log if you're going to actually fail on this line anyway

Reply:
actually looking at the code again for `expectErrorMessageToBe()` below we aren't actually enforcing the message... just doing a `expect(message).toBeTruthy()`... why is that?

<img width="759" alt="image" src="https://user-images.githubusercontent.com/1219069/219766225-af3aebf2-6f23-4ef9-8ff1-d0ffb45e1790.png">

Reply:
because none of the SDKs throw error messages that are the same, so rather than spending the time right now to fix them all we opted to just make sure there is an error in general and then log a warning if they don't match. Eventually we want to make them match but not right now

Reply:
ok then I guess I can scrap most of these changes then. need to figure out what else broke this then: https://github.com/DevCycleHQ/test-harness/actions/runs/4205682656/jobs/7298129382

Reply:
looks like something crashed the dotnet proxy server